gtfs_rt_validator_api.py

```diff
@@ -215,62 +268,94 @@ def validate_gcs_bucket_many(
 
     required_cols = ["gtfs_schedule_path", "gtfs_rt_glob_path"]
 
+    print(f"reading params from {param_csv}")
     fs = gcsfs.GCSFileSystem(project_id, token=token)
     params = pd.read_csv(fs.open(param_csv))
 
+    if limit:
+        logger.warn(f"limiting to {limit} rows")
+        params = params.iloc[:limit]
+
     # check that the parameters file has all required columns
     missing_cols = set(required_cols) - set(params.columns)
     if missing_cols:
         raise ValueError("parameter csv missing columns: %s" % missing_cols)
 
-    status = []
-    for idx, row in params.iterrows():
-        try:
-            validate_gcs_bucket(
+    statuses = []
+
+    print(f"processing {params.shape[0]} inputs with {threads} threads")
```

Comment:
Do we still want this to be a print statement? Or a logging one? (Noticed that it seems like we're trying to move to logging statements instead...?)

Reply:
yep! changed them all